### PR TITLE
Feature/structure

### PR DESCRIPTION
**Pull request by makitango**
Made too much in one commit, but so be it!

### CODE REVIEW

**Comment by CorneliusMoe** [comment]
Sieht gut aus 👍 👍

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          />
+          <figcaption>
+            A non-random picture, scrapped from my LinkedIn page.
+          </figcaption>
+        </figure>
+      </div>
+
+      <section id="contact">
+        <h3>Contact</h3>
+        <!--When clicking on these element, nothing happens! But I want to be redirected to the other websites. Why is it not working?-->
```
Comment (josch87) [inline index.html:37]: I don't know what you're talking about. Works on my machine.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        <a href="https://github.com/makitango">GitHub</a>
+        <a href="https://www.linkedin.com/in/dominic-bergmann-32825829/">LinkedIn</a>
+        <a href="https://codesandbox.io/u/makitango">CodeSandbox</a>
+      </section>
+
+      <section id="skills">
+        <h3>Skills</h3>
+        <!--This part could use some bulletpoint to be more structured-->
```
Comment (josch87) [inline index.html:45]: There are already bullet points!?

**Review by josch87 — APPROVED** [review]
LGTM

**Pull request merged**